src/lib/services/settings-service.ts

```diff
@@ -21,7 +21,7 @@ export const fetchSettings = async (request = fetch): Promise<Settings> => {
       options: settingsResponse?.Auth?.Options,
     },
     baseUrl:
-      import.meta?.env?.VITE_API ?? browser ? window.location.origin : '',
+      import.meta.env.VITE_API ?? (browser ? window.location.origin : ''),
```

Comment:
If this returns as an empty string instead of undefined, we will need || instead of ??

Reply:
`VITE_API=""` currently means a relative path the same as "/" https://github.com/temporalio/ui/blob/3ab14b473a34ef593c874673326493218726d664/src/lib/utilities/route-for-api.ts#L17